game/test/starter/GameTest.java

```diff
@@ -112,6 +109,7 @@ public void removeEntity() {
         Game.removeEntity(e1);
         Game.getDelayedEntitySet().update();
         assertFalse(Game.getEntities().contains(e1));
+        // 0, because empty after remove
         assertEquals(0, Game.getEntities().size());
```

Comment:
```suggestion
        // 0, because empty after remove
        assertEquals("The amount of entities should be 0", 0, Game.getEntities().size());
```